src/tests/playwright/pages/perf-gain.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script defer type="module">
+      import record from './processors/recorder/recorder-main.js';
+      import { beCloseTo, test, evaluate } from './util/audit.js';
+
+      test((async () => {
+        const ctx = new AudioContext({sampleRate: 48000});
+        const src = new ConstantSourceNode(ctx);
+        const gs = Array(100).fill().map(() => new GainNode(ctx));
+        const length = 1; // second
+        const {recorder, buffer} = await record(ctx, length);
+        gs.forEach((g, i) => i !== 0 && gs[i - 1].connect(g));
+        src.connect(gs[0]);
+        gs.at(-1).connect(recorder).connect(ctx.destination);
+
+        src.start();
+        src.stop(ctx.currentTime + 1);
+
+        const audioBuffer = await buffer;
+        const float32Buffer = audioBuffer.getChannelData(0);
+
+        await ctx.close();
+        return {buffer: float32Buffer};
+      })());
+
+      evaluate(async () => {
+        const bufferData = (await window.test).buffer;
```

Comment:
We have multiple `buffer`s in this file and they are not clear on what it does or means.